platform/ios/docs/guides/Adding Points to a Map.md

```diff
@@ -51,7 +51,6 @@ To use annotation views, implement `MGLMapViewDelegate` `-mapView:viewForAnnotat
 * No limit on style or image size
 * Full support for animations
 * Relative control over z-ordering using the `zPosition` property on `CALayer`
-* [Familiar API for MapKit users](https://www.mapbox.com/help/switch-mapkit/#annotations-pins)
```

Comment:
By the way, can we keep this pro in here but just delink it? It refers to the fact that the API is object-oriented and doesn’t have a steep learning curve.